spec/users/user_visit_menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "user visits menu", :type => :feature do
+  it "can see all items" do
+    user = create(:user)
+    item = create(:item)
+    visit '/menu'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.